plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -445,6 +448,38 @@ function executeCLI($scriptName, &$args) {
 		$this->usage($scriptName);
 	}
 
+	/**
+	 * Pull out getopt style long options.
+	 * @param &$args array
+	 * #param $optCodes array
+	 */
+	function parseOpts(&$args, $optCodes) {
+		$newArgs = [];
+		$opts = [];
+		$sticky = null;
+		foreach ($args as $arg) {
+			if ($sticky) {
+				$opts[$sticky] = $arg;
+				$sticky = null;
+				continue;
+			}
+			if (!starts_with($arg, '--')) {
```

Comment:
Is `starts_with` an actual function, or is that pseudocode?

Reply:
It is a helper provided by the laravel framework. I could take it out...

Reply:
Ah, OK, I just hadn't run across it. That's much more readable than the `substr` calls I've been relying on for the same thing.